src/main/java/me/ryanhamshire/GPFlags/flags/FlagDef_AllowVillagerTrade.java

```diff
@@ -0,0 +1,62 @@
+package me.ryanhamshire.GPFlags.flags;
+
+import me.ryanhamshire.GPFlags.Flag;
+import me.ryanhamshire.GPFlags.FlagManager;
+import me.ryanhamshire.GPFlags.GPFlags;
+import me.ryanhamshire.GPFlags.MessageSpecifier;
+import me.ryanhamshire.GPFlags.Messages;
+import me.ryanhamshire.GriefPrevention.events.ClaimPermissionCheckEvent;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.Villager;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.player.PlayerInteractEntityEvent;
+
+public class FlagDef_AllowVillagerTrade extends FlagDefinition {
+
+    public FlagDef_AllowVillagerTrade(FlagManager manager, GPFlags plugin) {
+        super(manager, plugin);
+    }
+
+    @Override
+    public String getName() {
+        return "AllowVillagerTrade";
+    }
+
+    @Override
+    public MessageSpecifier getSetMessage(String parameters) {
+        return new MessageSpecifier(Messages.EnabledAllowVillagerTrade);
+    }
+
+    @Override
+    public MessageSpecifier getUnSetMessage() {
+        return new MessageSpecifier(Messages.DisabledAllowVillagerTrade);
+    }
+    public boolean canTrade(Villager villager, Player player) {
```

Comment:
idk if we really need to modularize this since we only expect it to be called from one place

Reply:
TBH i dont think it matters, i just tend to code like that for readability